src/components/interaction-designer/block-editors/index.ts

```diff
@@ -2,8 +2,9 @@ export * from './choices'
 export * from './AdvancedExitEditor.vue'
 export * from './AdvancedExitsBuilder.vue'
 export * from './BlockEditor.vue'
-export * from './BlockId.vue'
+export {default as BlockId} from './BlockId.vue'
```

Comment:
!! Important: Marking a note here for future reference, in case we check this PR